.github/workflows/tests.yml

```diff
@@ -1,5 +1,7 @@
 name: Tests
-on: [push]
+on:
+  push:
+  pull_request: {}
```

Comment:
qq: is `{}` really needed?

Reply:
depends on the yaml parser! I think github doesnt require it, so i've removed.